airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/initialsync/MySqlInitialLoadHandler.java

```diff
@@ -83,7 +78,9 @@ public MySqlInitialLoadHandler(final JsonNode config,
   public List<AutoCloseableIterator<AirbyteMessage>> getIncrementalIterators(
                                                                              final ConfiguredAirbyteCatalog catalog,
                                                                              final Map<String, TableInfo<CommonField<MysqlType>>> tableNameToTable,
-                                                                             final Instant emittedAt) {
+                                                                             final Instant emittedAt,
+                                                                             final boolean decorateWithStartedStatus,
```

Comment:
nit: i feel we probably don't need these flags - unless it's designed for easily roll back. 

InitialLoadHandler are only used for RFR supported connectors (or certified connectors) anyway and for them we do need to support stream status

Reply:
It took my time to reason why this was needed:
* User cursor initial streams always need to be decorated with start and complete.
* CDC initial streams need to send start but not complete as CDC initial snapshot is followed by CDC session and only then we emit a complete status
* But Full refresh streams in CDC do need start and complete status